swift/apple/FirezoneKit/Sources/FirezoneKit/Models/Settings.swift

```diff
@@ -0,0 +1,97 @@
+//
+//  Settings.swift
+//  © 2025 Firezone, Inc.
+//  LICENSE: Apache-2.0
+//
+//  Settings represents the binding between our source-of-truth, Configuration, and user-configurable settings
+//  available in the SettingsView.
+
+import Foundation
+
+class Settings {
+  @Published var authURL: String
+  @Published var apiURL: String
+  @Published var logFilter: String
+  @Published var accountSlug: String
+  @Published var connectOnStart: Bool
+  var isAuthURLOverridden = false
+  var isApiURLOverridden = false
+  var isLogFilterOverridden = false
+  var isAccountSlugOverridden = false
+  var isConnectOnStartOverridden = false
+
+  private var configuration: Configuration
+
+  init(from configuration: Configuration) {
```

Comment:
What is this `from` keyword doing here? Never seen that before.

Reply:
It is simply naming the param